automation/check-patch.k8s-1.10.4-release.mounts

```diff
@@ -0,0 +1 @@
+check-patch.mounts
```

Comment:
What does the process look like when we update the cluster environments to a new cluster version?

Reply:
To add more environments we just have to add more check-patch.<env> files. Then add the kubevirtci image to the list and we will be set. Maybe even removing older versions if those are no longer tested.